packages/simple-hub/src/wallet/xstate-wallet-internals/index.ts

```diff
@@ -40,7 +40,7 @@ interface MixedAllocation {
 }
 
 type Allocation = SimpleAllocation | MixedAllocation;
-type Outcome = Allocation | SimpleGuarantee;
+export type Outcome = Allocation | SimpleGuarantee;
```

Comment:
This definition is at odds with the definition in `nitro-protocol` btw

https://github.com/statechannels/monorepo/blob/5955bb864c0e9e8d089e9d1099f3023e8f6c7a54/packages/nitro-protocol/src/contract/outcome.ts#L38-L39

Reply:
I think the shapes are genuinely different - the xstate wallet `Outcome` is a format that's more convenient to work with internally.